dynomitemanager-core/src/main/java/com/netflix/dynomitemanager/dynomite/DynomiteStandardTuner.java

```diff
@@ -117,6 +118,16 @@ public void writeAllProperties(String yamlLocation) throws IOException {
         entries.put("max_msgs", setMaxMsgs());
         entries.put("pem_key_file", floridaConfig.getDynomiteInstallDir() + "/conf/dynomite.pem");
 
+        if (!floridaConfig.getDynomiteHashtag().isEmpty()) {
+            if (floridaConfig.getDynomiteHashtag().length() != 2) {
+                logger.error("Hashtag must be of length 2. Provided hashtag: " + floridaConfig.getDynomiteHashtag()
+                        + " has length: " + floridaConfig.getDynomiteHashtag().length());
+                logger.error("Not setting any hashtag");
```

Comment:
should we start Dynomite without hashtag or just fail here? technically we should fail till the hashtab is solved, because it's an important configuration.

Reply:
I also thought about that and eventually decided that we can start Dynomite but without any hashtag. In that case, the client will not get any hashtag information from DM. The reason for this decision was that a misconfiguration of a fast property should not cause a production cluster to go south.

Reply:
You are right. I have added an exception to be thrown in case of a bad configuration.